src/core/lv_obj_tree.h

```diff
@@ -101,6 +101,14 @@ void lv_obj_move_foreground(struct _lv_obj_t * obj);
  */
 void lv_obj_move_background(struct _lv_obj_t * obj);
 
+/**
+ * Swap two objects within the same parent.
```

Comment:
It should work with different parents too.

Reply:
Thanks for the fixes. It seems only this comment should be updated.

Reply:
ok